src/apiDefs/env.ts

```diff
@@ -18,3 +18,19 @@ export const getEnvFlags = () => {
   }, {});
   return envFlags;
 };
+
+export const getCategoryFlags = () => {
+  const apiDefinitions = getApiDefinitions();
+  const categories = {};
+  getApiCategoryOrder().forEach((category) => {
+    categories[category] = apiDefinitions[category].apis.some( api => isHostedApiEnabled(api.urlFragment, api.enabledByDefault) );
+  });
+  return categories;
+};
+
+export const getApiCategories = () => {
+  const categoryFlags = getCategoryFlags();
+  return Object.keys(categoryFlags).filter((category) => {
```

Comment:
can inline this if you want:

```suggestion
  return Object.keys(categoryFlags).filter((category) => categoryFlags[category]);
```

another small thought that i have here is that it might be useful to use `getApiCategoryOrder()` rather than `Object.keys(categoryFlags)` here? that way this can operate as an analogous function that cuts down that array to the relevant categories, as you had originally written it.